packages/camera/camera_web/README.md

```diff
@@ -6,7 +6,10 @@ The web implementation of [`camera`][camera].
 
 ## Usage
 
-This package is [endorsed](https://flutter.dev/docs/development/packages-and-plugins/developing-packages#endorsed-federated-plugin), which means you can simply use `camera` normally. This package will be automatically included in your app when you do.
+### Depend on the package
+
+This package is not an endorsed implementation of the google_maps_flutter plugin yet,
```

Comment:
Optional nit: Maybe keep the link from the old version, but on "an endorsed implementation"?

Reply:
Done, I grabbed the text from another plugin that doesn't have the link (maps?), but it's a very good addition.